app/views/users/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<h1>Create a User</h1>
+
+<!-- <form action="/users" method="POST" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="username">Username:</label>
+  <input type="text" name="user[username]" id="username"><br><br>
+  <label for="email">Email:</label>
+  <input type="text" name="user[email]" id="email"><br><br>
+  <label for="username">Password:</label>
+  <input type="text" name="user[password]" id="password"><br><br>
+  <input type="submit" name="commit" value="Create User">
+</form> -->
+
+<!-- <%= form_tag('/users') do %>
+  <%= label_tag(:username, 'Username:') %>
+  <%= text_field_tag(:email) %><br><br>
```

Comment:
- [ ] This can be written this way
````ruby
  <%= label_tag 'username' %>
  <%= text_field_tag 'username' %>
`````
- [ ] In the text field tag it should be username not email